examples/farming_sim.rs

```diff
@@ -483,15 +481,15 @@ fn init_entities(
 
     commands.insert_resource(AmbientLight {
         color: Color::WHITE,
-        brightness: 1.0,
+        brightness: 700.0,
```

Comment:
I'm a little put off by this massive value, and same with `intensity`. You said that Bevy got some new lighting settings: is there some other setting on the camera or something that we need to adjust in order for `AmbientLight` and `Spotlight` to have reasonable-looking values? This just feels like it's papering over a different issue.

Reply:
Definitely agree with the weirdness. However I remember seeing some conversation about this in rendering-dev and consensus that the "realistic" values should be much higher

Examples in the bevy repo also seem to have similarly large values:

[animated_fox](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/animation/animated_fox.rs#L10) - ambient light intensity 2000.0

[alien cake addict](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/games/alien_cake_addict.rs#L118) - point light intensity 2,000,000

[3d_shapes](https://github.com/bevyengine/bevy/blob/a7aee1d03ac2070ae21b5f26253c0d8dc9d9af67/examples/3d/3d_shapes.rs#L70) - point light intensity 10,000,000

Reply:
that's so weird.

Ok, I guess.